bootstrap.php

```diff
@@ -43,9 +42,10 @@
 // Load Composer autoloader:
 require_once(dirname(__FILE__) . '/vendor/autoload.php');
 
+\PHPCI\ErrorHandler::register();
+
 if (defined('PHPCI_IS_CONSOLE') && PHPCI_IS_CONSOLE) {
     $loggerConfig = LoggerConfig::newFromFile(__DIR__ . "/loggerconfig.php");
```

Comment:
You have removed the Handler:register call, is the LoggerConfig::newFromFile stil needed because the var $loggerConfig isn't used in this file?